modules/core/src/lib/layer.js

```diff
@@ -363,13 +368,8 @@ export default class Layer extends Component {
     }
 
     if (name === 'all') {
-      log.log(LOG_PRIORITY_UPDATE, `updateTriggers invalidating all attributes: ${diffReason}`)();
```

Comment:
Aren't these pretty important? Assuming they are covered somewhere else?

Reply:
They are covered in AttributeManager.